recipes/nbcommands/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
Could you please remove all the comments?

Reply:
Yes, removing.